demos/JWST/S5_template.ecf

```diff
@@ -5,7 +5,7 @@ rescale_err     False    # Rescale uncertainties to have reduced chi-squared of
 fit_par         ./S5_fit_par_template.ecf # What fitting ecf do you want to use?
 run_verbose     True
 fit_method      [dynesty] #options are: lsq, emcee, dynesty (can list multiple types separated by commas)
-run_myfuncs     [transit,polynomial] #options are: transit, expramp, and polynomial (can list multiple types separated by commas)
+run_myfuncs     [transit,polynomial] #options are: transit, eclipse, phase, expramp, and polynomial (can list multiple types separated by commas)
```

Comment:
I'd rather use `phasecurve` since `phase` could be confused with orbital phase.  
This actually begs the deeper question: do we want to reference BATMAN in the transit, eclipse, and PC model names (e.g., batman_tr, batman_ecl, and batman_pc)?  In the near future, I foresee other transit model implementations that apply for sophisticated parameterizations to account for differences between ingress and egress.